scripts/migrate_to_marathon_task_tag_test.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/allegro/marathon-consul/consul"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMigrateToMarathonTaskTag_shouldMigrateOnlyServicesWithMarathonTagAndWithoutTaskTag(t *testing.T) {
```

Comment:
Maybe you want to use [testify suite](https://godoc.org/github.com/stretchr/testify/suite) here?